Userland/Services/WebDriver/Client.h

```diff
@@ -86,6 +87,9 @@ class Client final : public Web::WebDriver::Client {
     virtual Web::WebDriver::Response print_page(Web::WebDriver::Parameters parameters, JsonValue payload) override;
 
     static NonnullOwnPtrVector<Session> s_sessions;
+
+    /// https://w3c.github.io/webdriver/#dfn-maximum-active-sessions
+    static Optional<u32> s_active_sessions_limit;
```

Comment:
Can we initialize this here instead?

```suggestion
    static Optional<u32> s_active_sessions_limit { 1 };
```

Reply:
Why's this an `Optional` anyways?

Reply:
Also use only two forward slashes for the link comment please

Reply:
> Why's this an `Optional` anyways?

To accomodate for the potentially unlimited sessions. Which data structure would you suggest instead?

Reply:
> Can we initialize this here instead?

You get: "_Non-const static data member must be initialized out of line - clang(in_class_initializer_non_const)_"

Reply:
> To accomodate for the potentially unlimited sessions. Which data structure would you suggest instead?

If that's an option in the spec, and we have the ability to say "please run with unlimited sessions", then an `Optional` totally makes sense. Otherwise, as the code is right now, we just have an `Optional` that has always has a value, which is kinda weird.